File: CS/Sample/Controllers/HomeController.cs
using System;
using System.Web.Mvc;
using DevExpress.Web.Mvc;
using Sample.Models;

namespace Sample.Controllers {
    public class HomeController : Controller {
        PersonsList list = new PersonsList();

        public ActionResult Index() {
            return View(list.GetPersons());
        }

        public ActionResult GridViewPartial() {
            return PartialView(list.GetPersons());
        }

        public ActionResult About(int id) {
            ViewData["Key"] = id;
            return View();
        }
    }
}

File: VB/Sample/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc
Imports Sample.Models

Namespace Sample.Controllers
	Public Class HomeController
		Inherits Controller
		Private list As New PersonsList()

		Public Function Index() As ActionResult
			Return View(list.GetPersons())
		End Function

		Public Function GridViewPartial() As ActionResult
			Return PartialView(list.GetPersons())
		End Function

		Public Function About(ByVal id As Integer) As ActionResult
			ViewData("Key") = id
			Return View()
		End Function
	End Class
End Namespace